pkgs/applications/networking/instant-messengers/sky/default.nix

```diff
@@ -0,0 +1,76 @@
+{ stdenv, fetchurl, file, lib, libX11, libXScrnSaver,
+  gcc-unwrapped, libGL, qt5, SDL, libpulseaudio,
+  libXrandr, libXext, libXcursor, libXinerama, libXi,
+  ffmpeg-full, curl, sqlite, openssl_1_1_0,
+  libuuid, openh264, libv4l, libxkbfile, libXv, zlib, libXmu,
+  libXtst, libXdamage, pam, patchelfUnstable, libXfixes, libXrender, libjpeg_original
+}:
+ let
+   libjpeg_original_fix = libjpeg_original.overrideAttrs (oldAttrs: {
+    src = fetchurl{
+      url = http://www.ijg.org/files/jpegsrc.v8d.tar.gz;
+      sha256 = "1cz0dy05mgxqdgjf52p54yxpyy95rgl30cnazdrfmw7hfca9n0h0";
+    };
+  });
+in
+stdenv.mkDerivation rec {
+  version = "2.1.7300.1";
```

Comment:
If you split `2.1.7300` and `1` into separate variables, and interpolated them here, it would make construction of the URL much easier.

Reply:
You mean splitting it into version_major="2.1.7300" and version_minor="1"?

Reply:
Something like that, yeah